olimex/teres_i/default.nix

```diff
@@ -0,0 +1,30 @@
+{ config, lib, pkgs, ...}:
+
+{
+  hardware.deviceTree = {
+    name = lib.mkDefault "allwinner/sun50i-a64-teres-i.dts";
+    enable = lib.mkDefault true;
+  };
+
+  boot = {
+    consoleLogLevel = lib.mkDefault 7;
```

Comment:
Why change the default console log level?

Reply:
> Why change the default console log level? -- @samueldr (https://github.com/NixOS/nixos-hardware/pull/750#discussion_r1349257285)

[Other boards seems to change that so it's an attempt to follow the coding standard](https://github.com/search?q=repo%3ANixOS%2Fnixos-hardware%20%20consoleLogLevel%20%3D%20lib.mkDefault%207%3B&type=code)